RawWaterManagement/HydraulicSimulationResult/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "urn:ngsi-ld:dataModel:id:CWDU:80754394",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1999-04-07T01:11:20Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2007-04-19T05:09:16Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Moment knowledge where sea among role only. News prepare fact mother heavy. None music owner the."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Feeling pick almost century occur in understand. Somebody within page language."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Suffer half effort happy couple. Friend return audience determine. Either part economy me career five network."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Much practice require occur."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Rather I traditional born. Center song choose agreement."
+  },
+  "owner": {
```

Comment:
1. Relationships should not be modelled as properties, otherwise you completely lost the interest of NGSI-LD which is based on graph. 
2. Arrays in NGSI-LD are stored as strings so you can not use the API to query indiviudual alements in a array. 
3. Relationships can not be provided within arrays.

When you need to have several relationships with the same name, then you have to use the datasetId 

Example for a relationship where one entity would "monitor" 2 different rooms. It thus have 2 relationshipsnames "monitored" directed toward room01 and room02. The datasetId allows to differentiate the 2 relationships.
"monitors":[
	{
	   "type":"Relationship",
	   "object":"urn:ngsi-ld:Room:01",
	   "datasetId":"urn:ngsi-ld:dataSet:01"},
	},
	"type":"Relationship",
	"object":"urn:ngsi-ld:Room:02" ,
	"datasetId":"urn:ngsi-ld:dataSet:02"}
	]